src/Liip/RMT/VCS/Git.php

```diff
@@ -47,6 +47,10 @@ public function getTags()
 
     public function createTag($tagName)
     {
+        // this requires git and gpg configured
+        if (isset($this->options['sign-tag']) && $this->options['sign-tag']) {
+            return $this->executeGitCommand("tag -s $tagName -m $tagName");
+        }
```

Comment:
It's a minor styling comment, but personally I will not duplicate the `tag` instruction, but inserting a new sign variable, it will be easier for maintenance. Like this:

    $signOption = isset($this->options['sign-tag']) && $this->options['sign-tag']==true ? ' -s ' : '';
    return $this->executeGitCommand("tag $signOption $tagName -m $tagName");

Wdyt?